cmd/juju/model/exportbundle_test.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for info.
+
+package model_test
+
+import (
+	"github.com/juju/cmd/cmdtesting"
+	jujutesting "github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/cmd/juju/model"
+	coremodel "github.com/juju/juju/core/model"
+	"github.com/juju/juju/jujuclient"
+	"github.com/juju/juju/testing"
+)
+
+type ExportBundleCommandSuite struct {
+	testing.FakeJujuXDGDataHomeSuite
+	fake  *fakeExportBundleClient
+	stub  *jujutesting.Stub
+	store *jujuclient.MemStore
+}
+
+var _ = gc.Suite(&ExportBundleCommandSuite{})
+
+type fakeExportBundleClient struct {
+	*jujutesting.Stub
+	bestAPIVersion int
+}
+
+func (f *fakeExportBundleClient) Close() error { return nil }
+
+func (f *fakeExportBundleClient) ExportBundle() (string, error) {
+	f.MethodCall(f, "ExportBundle")
+	if err := f.NextErr(); err != nil {
+		return "", err
+	}
+	return "applications:\n" +
```

Comment:
You want this string to be a variable. You might also want to test what happens if it's empty?

Reply:
i have simplified this attr and also bestVersion as part of fakeExportBundleAPI struct so that each test sets the value.